Gemfile

```diff
@@ -13,9 +13,18 @@ end
 if ENV['CUCUMBER_RUBY_WIRE']
   gem 'cucumber-wire', path: ENV['CUCUMBER_RUBY_WIRE']
 elsif !ENV['CUCUMBER_USE_RELEASED_GEMS']
-  gem 'cucumber-wire', github: 'cucumber/cucumber-ruby-wire'
+  gem 'cucumber-wire', github: 'cucumber/cucumber-ruby-wire', branch: 'enable-circle-ci-windows-build'
 end
 
 gem 'cucumber-expressions', path: ENV['CUCUMBER_EXPRESSIONS_RUBY'] if ENV['CUCUMBER_EXPRESSIONS_RUBY']
 gem 'cucumber-messages', path: ENV['CUCUMBER_MESSAGES_RUBY'] if ENV['CUCUMBER_MESSAGES_RUBY']
 gem 'gherkin', path: ENV['GHERKIN_RUBY'] if ENV['GHERKIN_RUBY']
+
+require 'rbconfig'
+# rubocop:disable Style/DoubleNegation
+is_windows = !!(RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/)
+# rubocop:enable Style/DoubleNegation
+
+install_if -> { !is_windows } do
```

Comment:
does rubocop not like building on windows? I wasn't aware of any issues when I developed on windows using it.

Reply:
Rubocop relies on the [`json` gem](https://rubygems.org/gems/json/versions/2.0.3) (not the built-in one) and it causes issues when running `bundle install` on the Windows host of Circle CI (seem to work fine on AppVeyor that said).

As our goal is to have a tool as simple as possible to install, we're doing some cleanup in the gems to ensure we don't run into issues with gems that use native code.

In the case of Rubocop, as the rules are already executed 5 times (one for each supported Ruby), I think we can skip in on Windows :D

Reply:
I've no problem with that at all. But where does it rely on `json` I checked the deps and my local installs and I can't trace it.

Am I being an idiot?

Reply:
@vincent-psarga wasn't the problem with `rubocop` rather that it depends on `jaro_winkler`, which requires a C compiler in order to install?

Reply:
There was a PR to remove it - https://github.com/rubocop-hq/rubocop/pull/7673 but it's been rejected/postponed.

Reply:
yes indeed, my bad, got mixed between the various dependence failing to build there.

Reply:
All makes sense now, I have some experience of these with `curb` another native gem. No probs my side then